hikari/users.py

```diff
@@ -434,6 +508,11 @@ def app(self) -> traits.RESTAware:
     def avatar_hash(self) -> typing.Optional[str]:
```

Comment:
need to redefine accent_color and accent_colour here as well to narrow their return types down to Optional[Color]